docs/resources/schema.md

```diff
@@ -12,14 +12,38 @@ description: |-
 ## Example Usage
 
 ```terraform
+# basic resource
 resource "snowflake_schema" "schema" {
-  database = "database"
-  name     = "schema"
-  comment  = "A schema."
+  name     = "schema_name"
+  database = "database_name"
+}
+
+# resource with all fields set
+resource "snowflake_schema" "schema" {
+  name                = "schema_name"
+  database            = "database_name"
+  with_managed_access = true
+  is_transient        = true
+  comment             = "my schema"
+
+  data_retention_time_in_days                   = 10
```

Comment:
I believe this is an invalid config because the data retention time can only be 0 or 1 for transient objects. Make it not transient or make sure the values are correct.

Reply:
Right, fixed.